core/parentchain/indirect-calls-executor/src/indirect_calls_executor.rs

```diff
@@ -123,13 +123,19 @@ where
 	where
 		ParentchainBlock: ParentchainBlockTrait<Hash = H256>,
 	{
-		debug!("Scanning block {:?} for relevant xt", block.header().number());
+		let block_number = *block.header().number();
+		debug!("Scanning block {:?} for relevant xt", block_number);
 		let mut executed_shielding_calls = Vec::<H256>::new();
 		for xt_opaque in block.extrinsics().iter() {
+			let mut shield_funds_decoded = false;
```

Comment:
Should this go into master?

Reply:
hmm first I thought yes, but I can probably remove it again to reduce clutter in this function. There is a comment at least that documents the behavior.